detox/ios/DetoxXCUITestRunner/DetoxXCUITestRunner/Extensions/XCUIApplication+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  XCUIApplication+Extensions.swift (DetoxXCUITestRunner)
+//  Created by Asaf Korem (Wix.com) on 2024.
+//
+
+import Foundation
+import XCTest
+
+extension XCUIApplication {
+  static var springboard: XCUIApplication {
+    return XCUIApplication(bundleIdentifier: "com.apple.springboard")
+  }
+
+  static var safari: XCUIApplication {
+    return XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+  }
```

Comment:
Dead code?

Reply:
lol, yeah.. will revert and return this when implementing safari support